packages/babel-preset-default/bin/index.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
```

Comment:
@gziolo: As much as I love [shebangs](https://en.wikipedia.org/wiki/Shebang_(Unix)), AFAIK they just don't work well or at all with Windows. This just emerged in #32329. I'm wondering if we should/could forbid it with our linter — wdyt?

Reply:
I copied it over from other scripts. What would be the alternative? If it causes issues we should teach ESLint to prevent issues.

Reply:
The alternative would be to just not have any shebang, meaning that scripts would just have code:

```js
1. /**
2.  * External dependencies
3.  */
4. const builder = ...;
```

This would make it impossible for anyone in a UNIX system to invoke `./index.js` (they would have to explicitly call the interpreter with `node index.js`), thus eliminating the possibility of merging something that would break in Windows.

Reply:
Sounds good, we can live with it 👍🏻

Reply:
Filed: https://github.com/WordPress/gutenberg/issues/32504.